apps/condo/domains/user/utils/serverSchema/guards.js

```diff
@@ -37,6 +42,30 @@ class RedisGuard {
         }
     }
 
+    async checkPaymentLinkLimitCounters (rawIp) {
```

Comment:
can we use it in more abstract way. to avoid abstraction leak inside the common domain

Reply:
Split up business logic for limits & guard common limit methods